### PR TITLE
mkcloud: DO NOT MERGE THIS FAKE PULL REQUEST

### PR DESCRIPTION
**Pull request by jdsn**
its just for testing - DO NOT MERGE

### CODE REVIEW

File: scripts/DO-NOT-MERGE-THIS-PR [hunk 1]
```diff
@@ -0,0 +1 @@
+FAKE-PULLREQUEST-DO-NOT-MERGE
```
Comment (aspiers) [inline scripts/DO-NOT-MERGE-THIS-PR]: -1, please can we rewrite this in Go ;-)

**Comment by MaximilianMeister** [comment]
-1 please provide unittest

File: scripts/DO-NOT-MERGE-THIS-PR (hunk 1 above)
Comment (mimi1vx) [inline scripts/DO-NOT-MERGE-THIS-PR]: Go ? noooo

Comment (jdsn) [inline scripts/DO-NOT-MERGE-THIS-PR]: I think we should schedule a Go-NoGo meeting :)

**Comment by jdsn** [comment]
Still no unittests but I added a bar. It should be possible to order drinks now.

File: scripts/DO-NOT-MERGE-THIS-PR (hunk 1 above)
Comment (tboerger) [inline scripts/DO-NOT-MERGE-THIS-PR]: I have already started to rewrite openstack in go.

**Comment by MaximilianMeister** [comment]
> Still no unittests but I added a bar. It should be possible to order drinks now.

LBTM

i tried to order a drink, but i got nothing. so i went to the crowbar

**Comment by jdsn** [comment]
This was the first automatic mkcloud self-gating run.

**Comment by aspiers** [comment]
Woooo! :)

**Comment by jdsn** [comment]
Now rebuilding all open PRs. All are pending.

Unfortunately we only have one mkcloud worker currently as the github-status.rb does not work on SLE11SP3. But we wanted to migrate the 2nd host anyway.

**Comment by tboerger** [comment]
> Unfortunately we only have one mkcloud worker currently as the github-status.rb does not work on SLE11SP3. But we wanted to migrate the 2nd host anyway.

Why does it not work there?

**Comment by jdsn** [comment]
@tboerger current octokit requires ruby > 1.9.$something , and with older octokit versions various errors raise. A we wanted to update the node anyway, I think we should not invest much time here.

**Pull request closed without merging**